packages/overlay/src/integrations/sentry/components/performance/TransactionsList.tsx

```diff
@@ -0,0 +1,149 @@
+import { useMemo, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { ReactComponent as Sort } from '~/assets/sort.svg';
+import { ReactComponent as SortDown } from '~/assets/sortDown.svg';
+import classNames from '~/lib/classNames';
+import { TRANSACTIONS_SORT_KEYS, TRANSACTIONS_TABLE_HEADERS } from '../../constants';
+import { useSentryEvents } from '../../data/useSentryEvents';
+import { useSentryHelpers } from '../../data/useSentryHelpers';
+import { SentryTransactionEvent } from '../../types';
+import DateTime from '../DateTime';
+
+export default function TransactionsList({ showAll }: { showAll: boolean }) {
+  const events = useSentryEvents();
+  const helpers = useSentryHelpers();
+  const navigate = useNavigate();
+
+  const [sort, setSort] = useState({
+    active: '',
+    asc: false,
+  });
+
+  const toggleSortOrder = (type: string) =>
+    setSort(prev =>
+      prev.active === type
+        ? {
+            active: type,
+            asc: !prev.asc,
+          }
+        : {
+            active: type,
+            asc: false,
+          },
+    );
+
+  type GroupedTransactionsValue = {
+    transactions: SentryTransactionEvent[];
+    lastSeen: number;
+  };
+  type GroupedTransactions = Record<string, GroupedTransactionsValue>;
+
+  type TransactionsInfoComparator = (
+    a: [string, GroupedTransactionsValue],
+    b: [string, GroupedTransactionsValue],
+  ) => number;
+  type TransactionsSortTypes = (typeof TRANSACTIONS_SORT_KEYS)[keyof typeof TRANSACTIONS_SORT_KEYS];
+
+  const transactionsList: [string, GroupedTransactionsValue][] = useMemo(() => {
+    const COMPARATORS: Record<TransactionsSortTypes, TransactionsInfoComparator> = {
+      [TRANSACTIONS_SORT_KEYS.count]: (a, b) => a[1].transactions.length - b[1].transactions.length,
+      [TRANSACTIONS_SORT_KEYS.count]: (a, b) => a[1].lastSeen - b[1].lastSeen,
```

Comment:
Pretty sure keys are incorrect here. Guessing the second one should be `lastSeen` or something?
```suggestion
      [TRANSACTIONS_SORT_KEYS.count]: (a, b) => a[1].transactions.length - b[1].transactions.length,
      [TRANSACTIONS_SORT_KEYS.lastSeen]: (a, b) => a[1].lastSeen - b[1].lastSeen,
```